src/ToricVarieties/LineBundles/auxilliary.jl

```diff
@@ -0,0 +1,34 @@
+
+function command_string(v::AbstractNormalToricVariety, c::Vector{fmpz})
+    # Initialize a list of strings, which we will eventually join to form the command string
+    string_list = Vector{String}()
+    
+    # Define helper function
+    joincomma(list) = join([string(x) for x in list], ", ")
+    
+    # cohomCalg cannot handle "[" nor "]" in coordinate names.
+    # -> Remove these characters both in names of homogeneous variables and the generators of the Stanley-Reisner ideal.
+    
+    # Extract homogeneous variables
+    vars = [replace(replace(string(x), "[" => ""), "]" => "") for x in Hecke.gens(cox_ring(v))]
+    if length(unique(vars)) != length(vars)
+        # consistency check: there should be no duplicates in vars
+        throw(ArgumentError("Duplicate variable names after removing [ and ]."))
+    end
+    
+    # Add information about grading of Cox ring to string_list
+    divisors = gens(torusinvariant_divisor_group(v))
+    for i in 1:length(vars)
+        tmp = joincomma([string(x) for x in map_from_weil_divisors_to_class_group(v)(divisors[i]).coeff])
+        push!(string_list, "vertex $(vars[i]) | GLSM: ( $(tmp) )")
+    end
+    
+    # Add information about the Stanley-Reisner ideal to string_list
+    push!(string_list, "srideal [" * replace(replace(joincomma([string(x) for x in gens(stanley_reisner_ideal(v))]), "[" => ""), "]" => "") * "]" )
```

Comment:
You already invoke `string` in `joincomma`, so you can simplify this (also elsewhere)
```suggestion
    push!(string_list, "srideal [" * replace(replace(joincomma(gens(stanley_reisner_ideal(v))), "[" => ""), "]" => "") * "]" )
```

Reply:
Thank you. Done.